app/util.py

```diff
@@ -75,6 +75,14 @@ def fetch_server_status(id):
 
 	return d
 
+@cached(cache=TTLCache(ttl=5, maxsize=10))
```

Comment:
Up the ttl to 10 like the rest

Reply:
The `status` TTL literally immediately above this one is also 5.